src/main/scala/org/constellation/p2p/PeerAPI.scala

```diff
@@ -119,6 +119,7 @@ class PeerAPI(override val ipManager: IPManager[IO])(
             APIDirective.handle(
               dao.snapshotBroadcastService.getRecentSnapshots
             ) { result =>
+              logger.debug(s"snapshot received ${s}")
```

Comment:
are you sure we'd like to log the full snapshot instead of the hash?

Reply:
This object consist of 2 fields `case class SnapshotCreated(hash: String, height: Long)` I don't think is much